### DISCUSSION TITLE
How can I upgrade to v0.11 from v0.6.4?

### DISCUSSION THREAD

**Discussion by pedrohba1** [wevm/wagmi#1771] [Q&A]
I'm currently having a hard time updating directly to version 0.11. Maybe upgrading to v0.9 will be easier, but I would like a note of advice.

I'm having problems with nearly all ABI that worked before:

![Screenshot from 2023-02-01 15-09-26](https://user-images.githubusercontent.com/39497156/216127051-c97773e6-3183-44f2-b801-f23e2c4c5b56.png)
![Screenshot from 2023-02-01 15-09-14](https://user-images.githubusercontent.com/39497156/216127063-9ae1d32b-9909-47c3-a920-837e677f662d.png)

Also, this one is kinda weird:

![Screenshot from 2023-02-01 15-10-25](https://user-images.githubusercontent.com/39497156/216127208-4d515787-470f-4e1a-8c09-cb6bd2eecc64.png)

```

The inferred type of 'wagmiClient' cannot be named without a reference to '.pnpm/@wagmi+core@0.9.2_mqau3ypy7zv33pmr7tiecicv2q/node_modules/@wagmi/core'. 
This is likely not portable. A type annotation is necessary.ts(2742)

```

Any advice on fixing these errors? I saw that it's necessary to update typescript in order to properly use wagmi 0.11.
However, I did the update and these still seem to be breaking.

**Comment by tmm** [comment]
Check out the migration guide: https://wagmi.sh/react/migration-guide
